packages/react-charting/docs/colors.md

```diff
@@ -0,0 +1,37 @@
+# Colors
+
+The color prop is currently optional for curves and lines in AreaChart and LineChart, respectively. If color is not provided, the chart elements will be rendered with colors from the default palette. The users can provide custom colors or use colors from these palettes:
```

Comment:
Specify here clearly that we provide 3 ways to specify the display color for each data series.
1. Default qualitative palette for LineChart and AreaChart if no color property is specified.
2. Ability to specify a different color from the Qualitative or Semantic palette.
3. Specify any other hex code. In this case the user is responsible for maintaining the accessibility contrast ratio between adjacent data series.

And then you can talk in more details about each option and specify examples.

Reply:
done